### DISCUSSION TITLE
Payment sheet does not show up in Android release build

### DISCUSSION THREAD

**Discussion by zamargab** [flutter-stripe/flutter_stripe#1704] [Q&A]
**Describe the bug**
I have followed all the integration guidelines for Android including the proguard-rules and i tested on debug mode and everything worked fine but on building a release APK, I noticed the Payment sheet refused to come up. 

**To Reproduce**
1. Integrated package on Android platform
2. Create a payment and display payment sheet
3. Build release apk

**Expected behavior**
Payment sheet comes up and works fine in debug mode but refused to come up in release mode

**Smartphone / tablet**
 - Samsung A23
 - OS: Android 14
 - Package version: 10.1.1 and 9.5.0+1
 - Flutter version 3.13.9

**Comment by AditiFlutter** [comment]
i am also facing this issue u get any solution .

**Comment by AbdullahChauhan** [comment]
@zamargab @AditiFlutter 
Make sure you followed all the steps for Android configuration. I'm also facing the same issue and found that I missed adding the `Theme.MaterialComponents` in the `styles.xml` file (values-night directory).

**Comment by zamargab** [comment]
I copied the styles.xml in the example project and the problem persists for Android. I even tried building for IOS and the same issue exists in IOS

**Comment by AbdullahChauhan** [comment]
@zamargab 
If payment sheet is not opening successfully, there has to be the configuration issue. Please check again if you've fulfilled all the steps correctly for both Android & iOS.

**Comment by remonh87** [comment]
I would check https://docs.page/flutter-stripe/flutter_stripe/sheet if you missed something
